### PR TITLE
Dockerfile fix

### PR DESCRIPTION
**Pull request by JaydeepGo**
There was a problem with Dockerfile update and upgrade command, fixed in this patch.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,8 @@ RUN find /etc/systemd/system \
 RUN systemctl set-default multi-user.target
 CMD ["/sbin/init"]
 
-RUN apt-get update; apt-get -y upgrade
-RUN apt-get -y install monit apache2
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get -y install monit 
+RUN apt install -y apache2
```
Comment (dashohoxha) [inline Dockerfile:16]: Actually these changes are almost equivalent to the original. So they don't improve anything. The only effect of making these changes is that the cache is invalidated and that part of the image is built again.

**Review by dashohoxha — COMMENTED** [review]
This patch does not change anything, but anyway thanks for trying to fix the problem.

**Comment by JaydeepGo** [comment]
But sir if we change given lines, never get 404 maybe.
Isn't it right?

**Comment by dashohoxha** [comment]
Why do you think so?
What is the problem in the old lines that is fixed by your changes?
They are equivalent (almost).

**Comment by JaydeepGo** [comment]
Sorry for the trouble and both are indeed almost same.
It just other one solve the problem (not really) then I think both are different but after the whole checking understands your point.
Thanks for help.

**Comment by dashohoxha** [comment]
> Sorry for the trouble

It is not a trouble at all. You had a problem with docker and you tried to fix it. That's Ok.

**Pull request closed without merging**